### PR TITLE
jinjibu.jp

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/112534

<details><summary>Screenshot:</summary>

Снимок экрана 2022-03-21 в 14 23 40

</details><br/>

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -59,6 +59,7 @@ note.linemusic.jp,diehardtales.com,note.com,note.kagg.jp,note.jp,note.yamap.com#
...
 !
 fakty.tvn24.pl##.main-social
+jinjibu.jp##.colored > li:not([class])
```
Comment (Yuki2718) [inline SocialFilter/sections/specific.txt:62]: This too, should be `display: none !important;` and moved to general_extensions with `jinjibu.jp#?#.colored > p:contains(share)`.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply SocialFilter/sections/specific.txt:62]: @Yuki2718 `#?#` rules we add to the `css_extended.txt`.

Comment (Yuki2718) [reply SocialFilter/sections/specific.txt]: I mean
```
jinjibu.jp#$#.colored > li:not([class]) { display: none !important; }
jinjibu.jp#$?$#.colored > p:contains(share)  { display: none !important; }
```

**Review by zloyden — APPROVED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (Yuki2718) [reply SocialFilter/sections/specific.txt]: If `#?#.colored > p:contains(share)` is not applied (maybe on iPad, as apparently not a matter on smartphone), user see the share text left.

**Review by Yuki2718 — APPROVED** [review]

**Pull request merged**